### ISSUE TITLE
Google pay not working in Android

### ISSUE THREAD

**Issue by akhilsanker** [A-Tokyo/react-native-stripe-checkout-webview#55]
Google pay is not working in android device. I have tried react-native-webview/react-native-webview#920 (comment) this solution but no effect in android.

Stripe preview is working well in chrome with in the same android device. Is there anything I am missing, even from Stripe dashboard.

Please find my code below:

```
<StripeCheckout
stripePublicKey={STRIPE_PUBLISH_KEY.PUBLISH_KEY}
checkoutSessionInput={{
  sessionId: checkoutId
}}
onSuccess={() => {
  paymentSuccess();
}}
onCancel={() => {
  navigation.goBack();
}}
options={{
  htmlContentLoading: '',
  htmlContentHead: ''
}}
renderOnComplete={() => <></>}
onNavigationStateChange={() => {
  setIsPaymentSuccess(true);
}}
/>
```

Versions using:
react-native: 0.64.0
react-native-webview: 11.0.3
react-native-stripe-checkout-webview": 0.0.12,

Thanks

**Comment by hussainimdad004** [comment]
Having same issue

**Comment by hussainimdad004** [comment]
@A-Tokyo yes it works on normal browser. even in mobile chrome browser

**Comment by A-Tokyo** [comment]
I see this is an issue with react-native-webview 
- https://github.com/react-native-webview/react-native-webview/issues/1976
- https://stackoverflow.com/questions/66003027/react-native-webview-stripe-google-pay 

I think we should discuss there as the fix needs to be done from webview's side

**Comment by A-Tokyo** [comment]
@hussainimdad004 Check this out for apple pay https://github.com/react-native-webview/react-native-webview/issues/920#issuecomment-810866297

**Comment by Qiuzhuang** [comment]
> @hussainimdad004 Check this out for apple pay [react-native-webview/react-native-webview#920 (comment)](https://github.com/react-native-webview/react-native-webview/issues/920#issuecomment-810866297)

This seems to fix ios apple pay but not android google pay. Can we update the doc and file new issue for android side?

**Comment by iconico** [comment]
Has there been any updates on this issue?

**Comment by akhilsanker** [comment]
Hi @A-Tokyo ,

Any update on this ?

Thanks

**Comment by A-Tokyo** [comment]
Hi!

We're still waiting on react-native-web-view https://github.com/react-native-webview/react-native-webview/issues/1976 There's also a bounty on SF https://stackoverflow.com/questions/66003027/react-native-webview-stripe-google-pay
